### PR TITLE
plan review for MP Metrics 5.1

### PR DESCRIPTION
**Pull request by donbourne**

### CODE REVIEW

File: metrics/5.1/README.adoc [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+|----------------------------------------|-----------------------------|------
+| Summers Pittman, Vincent Mayers        | Atlanta JUG                 |  
+| Emily Jiang, Nathan Rauh               | IBM                         |  
+| Ed Bratt, Dmitry Kornilov              | Oracle                      |  
+| John Clingan, Roberto Cortez           | Red Hat                     |  
+| David Blevins, Jean-Louis Monteiro     | Tomitribe                   |  
+| Chandra Guntur, Michael Redlich        | Garden State Java User Group|  
+| Jan Westerkamp, Heiko W. Rupp          | iJUG                        |  
+| Kenji Kazumura, Takahiro Nagao         | Fujitsu                     |  
+| Reza Rahman, Edward Burns              | Microsoft                   |
```
Comment (Emily-Jiang) [inline metrics/5.1/README.adoc:48]: need to add Payara and Primeton to the list

Comment (Emily-Jiang) [reply metrics/5.1/README.adoc:48]: You can copy the list from [here](https://github.com/microprofile/microprofile-wg/tree/main/microprofile-7.0)

Comment (donbourne) [reply metrics/5.1/README.adoc:50]: thank you -- fixed

**Comment by Emily-Jiang** [comment]
@donbourne please add the ballot result to your PR

**Comment by donbourne** [comment]
@Emily-Jiang , I've added results from the ballot to the PR.  ready to merge?

**Review by Emily-Jiang — APPROVED** [review]

**Pull request merged**